pkg/cmd/run.go

```diff
@@ -189,6 +189,9 @@ doppler run --mount secrets.json -- cat secrets.json`,
 				} else if strings.HasSuffix(mountPath, ".env") {
 					mountFormat = models.EnvMountFormat
 					utils.LogDebug(fmt.Sprintf("Detected %s format", mountFormat.Type))
+				} else if strings.HasSuffix(mountPath, "appSettings.Development.json") || strings.HasSuffix(mountPath, "appSettings.json") {
```

Comment:
Would it make sense to do a regex check here?
https://go.dev/play/p/cjDrjqRTQ5Z

I mainly mention it because the .NET docs have some of their examples in lowercase or with other environments:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0

Reply:
Great catch! Will use regex instead.